csp-tech-radar.csv

```diff
@@ -8,3 +8,40 @@ Long lived branches with Gitflow,hold,techniques,TRUE,"Gitflow is a strict branc
 Microservices Envy,hold,techniques,TRUE,"Microservices can offer significant advantages to organizations, in terms of improving team autonomy and faster frequency of change. The additional complexity that comes from distributed systems requires an additional level of maturity and investment. We are concerned that some teams are rushing into adopting microservices without understanding the changes to development, test, and operations that are required to do them well. Our general advice remains simple. Avoid microservice envy and start with one or two services before rushing headlong into developing more, to allow your team's time to adjust and understand the right level of granularity."
 Testing as a separate team,hold,techniques,TRUE,"Fast feedback is a core tenet of Agile and critical to the success of a project. Using a separate QA team slows down this feedback, creates an ""us and them"" mentality and makes it more difficult to build quality into the software. Testing should be a tightly integrated activity and isn't something the team can outsource. We recommend integrated teams where testers work closely with developers instead of having testing as a separate organization."
 Anemic REST,hold,techniques,TRUE,"Failure to properly model the domain as a set of resources. Naively developing services that simply expose static, hierarchical data models via templated URLs result in an anemic REST implementation. In a richly modelled domain, REST should enable more than simple repetitive data fetching. In a fully evolved RESTful architecture, business events and abstract concepts are also modelled as resources, and the implementation should make effective use of hypertext, link relations and media types to maximize decoupling between services"
+Python,adopt,tools,FALSE,"As integration test on hold instead we can use Spock, but available for unit tests for projects Python.<br /><br /><p><strong>Teams: </strong>Wrights, Olympus, Krakatoa, Everest, Rockies, Vestrahorn</p>"
```

Comment:
this should one for languages and frameworks (I will do it as part of mine)
```suggestion
```

Reply:
Then I remove this line completely.